File: API/Hu.API/Hu.API/Models/TournamentModel.cs
using Newtonsoft.Json;

namespace Hu.API.Models
{
    public class TournamentModel
    {
        [JsonProperty("name")]
        public string? Name { get; set; }

        [JsonProperty("slug")]
        public string? Slug { get; set; }

        [JsonProperty("category")]
        public CategoryModel Category { get; set; } = new CategoryModel();

        [JsonProperty("uniqueTournament")]
        public UniqueTournamentModel UniqueTournament { get; set; } = new UniqueTournamentModel();

        [JsonProperty("priority")]
        public int Priority { get; set; }

        [JsonProperty("id")]
        public int Id { get; set; }
    }
}

File: UI/Hu/src/app/events/models/tournament.model.ts
import { Category } from "./category.model";
import { UniqueTournament } from "./unique-tournament.model";

export interface Tournament {
    name:              string;
    slug:              string;
    category:          Category;
    uniqueTournament?: UniqueTournament;
    priority:          number;
    isLive:            boolean;
    id:                number;
}